### DISCUSSION TITLE
Jori Lallo -- Linear

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#118] [Announcements]
## Background

- [Jori's Twitter](https://twitter.com/jorilallo)
- [Linear's Twitter](https://twitter.com/linear)
- [linear.app](https://linear.app/)

## Themes

- Local first software
- Issue tracking
- Linear Method

## Intro

**Andrew**: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

**Justin**: Hey everyone! Our guest today is Jori Lallo. Jori is one of the founders of Linear which is an absolutely delightful issue tracking and project management software. Jori, it's an absolute pleasure to have you on and I'm so excited to talk about Linear. Before we start, would you like to tell our listeners a bit more about yourself?

### Jori

- [x] Was Linear the first startup you've founded? If not, how is it different from your other experiences?
- [x] You were an early engineer at Coinbase. How did that experience shape your approach to how you do work at Linear?
- [x] You started linear with two friends, Tuomas and Karri. How did y'all meet and how has your relationship changed as Linear has grown?

### Linear the product

- [x] Issue tracking and project management is often an unsexy topic. Tools in this space tend to become the bane of engineer's lives. Why did y'all decide to tackle this problem? (What were the problems in other products you were trying to fix?)
- [x] Linear has a local first approach where users can install an app that auto syncs changes and can be used offline. Why was local first important enough to the team to have it a baseline feature? How has it scaled as the product has grown?
- [x] Linear is built with keyboard shortcuts as a primary means of interacting with the product. What is your product philosophy when it comes to build new features keyboard first?
- [ ] ~What is the most creative thing you’ve seen done with the API?~
- [x] How do you keep linear so fast?
- [ ] ~What’s the one killer features of linear you think makes it stand above the rest? (Or your favorite feature)~
- [x] How does linear help scale the experience to multiple teams?
- [ ] ~What are integrations? How do they make linear more useful?~

### Linear the company

- [x] You have this concept called the _Linear method_. Is that mostly how Linear approaches product development or how you encourage everyone who uses linear to approach product development?
- [ ] ~How do you handle hiring at Linear?~
- [ ] ~What are the aspects of Linear's culture that you think are unique and help you ship a better product?~

### Future

- [ ] What are you building towards? What is linear going to look like in 5 years?
- [ ] What do you think the future of devtools looks like?

**Comment by just-be-dev** [comment]
Tooltips:
- https://tana.inc/
- [Open Transclude for Networked Writing](https://subpixel.space/entries/open-transclude/) by Toby Shorin

**Comment by hipstersmoothie** [comment]
https://github.com/missive/emoji-mart
https://www.codeedit.app

**Comment by jorilallo** [comment]
https://www.amazon.com/CANARY-Corrugated-Cardboard-Fluorine-DC-190F/dp/B008RIS0UY/
https://www.festoolusa.com/products/drilling-and-screwdriving/cordless-drills/576099---cxs-2,6-set-us

**Comment by jorilallo** [comment]
Tech talk by Tuomas about Linear's Sync Engine https://youtu.be/WxK11RsLqp4?t=2171
